src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/ApplicationEntity.kt

```diff
@@ -356,7 +356,7 @@ class ApprovedPremisesApplicationEntity(
       requiredQualifications += UserQualification.WOMENS
     }
 
-    if (isEmergencyApplication == true) {
+    if (noticeType == Cas1ApplicationTimelinessCategory.emergency || noticeType == Cas1ApplicationTimelinessCategory.shortNotice) {
```

Comment:
Is it correct that shortNotice results in an EMERGENCY user qualification? If yes, maybe the EMERGENCY qualification needs renaming?

Reply:
Yeah, I did think about that, but we don't _really_ use this at the mo, as allocations are done via config